pkg/pillar/cmd/zedmanager/handlevolumemgr.go

```diff
@@ -303,6 +304,9 @@ func reactToSnapshotRollback(ctx *zedmanagerContext, volumesSnapshotStatus types
 	unpublishLocalAppInstanceConfig(ctx, appInstanceStatus.Key())
 	publishAppInstanceStatus(ctx, appInstanceStatus)
 	doUpdate(ctx, *config, appInstanceStatus)
+	// Mark the app as active locally to ensure it immediately restarts after rollback from a snapshot.
+	// This avoids delays in app activation that can occur due to edge app priority scheduling.
+	activeapp.CreateLocalAppActiveFile(log, appInstanceStatus.UUIDandVersion.UUID.String())
```

Comment:
How will it work if the applictaion was not originally set as Active? Will it sill forcefully start the app?